studio/src/components/feature-flags-table.tsx

```diff
@@ -0,0 +1,196 @@
+import { useUser } from "@/hooks/use-user";
+import { docsBaseURL } from "@/lib/constants";
+import { CommandLineIcon } from "@heroicons/react/24/outline";
+import { TooltipContent, TooltipTrigger } from "@radix-ui/react-tooltip";
+import {
+  FeatureFlag,
+  FederatedGraph,
+} from "@wundergraph/cosmo-connect/dist/platform/v1/platform_pb";
+import { formatDistanceToNow } from "date-fns";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { EmptyState } from "./empty-state";
+import { Badge } from "./ui/badge";
+import { Button } from "./ui/button";
+import { CLISteps } from "./ui/cli";
+import { Pagination } from "./ui/pagination";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+  TableWrapper,
+} from "./ui/table";
+import { Tooltip } from "./ui/tooltip";
+import { update } from "lodash";
+
+export const Empty = ({ graph }: { graph?: FederatedGraph }) => {
+  const router = useRouter();
+
+  let label = "[labels...]";
+  if (graph?.labelMatchers && graph.labelMatchers.length > 0) {
+    label = graph.labelMatchers[0].split(",")[0];
+  }
+  return (
+    <EmptyState
+      icon={<CommandLineIcon />}
+      title="Create feature flag using CLI"
+      description={
+        <>
+          No feature flags found. Use the CLI tool to create one.{" "}
+          <a
+            target="_blank"
+            rel="noreferrer"
+            href={docsBaseURL + "/cli/feature-flags/create"}
+            className="text-primary"
+          >
+            Learn more.
+          </a>
+        </>
+      }
+      actions={
+        <CLISteps
+          steps={[
+            {
+              description:
+                "Create a feature subgraphs using the below command.",
+              command: `npx wgc feature-subgraph create <feature-subgraph-name> --namespace ${router.query.namespace} -r <routing-url> --subgraph <base-subgraph-name>`,
+            },
+            {
+              description:
+                "Publish a feature subgraphs using the below command.",
+              command: `npx wgc subgraph publish <feature-subgraph-name> --namespace ${router.query.namespace} --schema <schema-path> `,
+            },
+            {
+              description: "Create a feature flag using the below command.",
+              command: `npx wgc feature-flag create <feature-flag-name> --namespace ${router.query.namespace} --label ${label} --feature-subgraphs <feature-subgraph-names...> -e`,
```

Comment:
The fourth step is to enable the feature flag.

Reply:
We enable the feature flag while creating it itself

Reply:
Can we use the full flag name? I think `--enabled` is easier to understand for beginners.

Reply:
Done